### PR TITLE
Created Flash messages - Mustafe

### PR DESCRIPTION
**Pull request by Muste8**
### Changes
Give a short summary of the changes made in this PR. 
-Created a partial _flashes.html.erb in views/layouts
-Creating a conditional styling for the different flash messages (SUCCESS, ERROR, WARNING, INFO)
-Added icons folder(flashes) in assets/images/
-Attached appropriate icon to each flash message

### Link to Issue 
Link Asana ticket here!
https://app.asana.com/0/1206510424537042/1206673931535433/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 
-It didn't mention this in the task, so I added the folder for the icons. 

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!
Screenshot 2024-03-26 at 10 10 23 AM
Screenshot 2024-03-26 at 11 06 03 AM
Screenshot 2024-03-26 at 11 32 08 AM
Screenshot 2024-03-26 at 11 08 14 AM
Screenshot 2024-03-26 at 11 33 09 AM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

**Review by camckin10 — APPROVED** [review]
This looks great!

File: app/views/layouts/_flashes.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                  'blue'
+                end %>
+  <div class="fixed right-0 top-0 mx-2 my-2 w-2/5">
+    <!-- Narrow dark box on top -->
+    <div class="h-2 rounded-t bg-<%= color %>-500"></div>
+
+    <!-- Bottom box for message -->
+    <div class="rounded-b bg-<%= color %>-300 text-black flex flex-wrap items-center justify-between">
+      <div class="flex flex-wrap p-4 items-center">
+        <img src="<%= asset_path('flashes/' + name + '.svg') %>" alt="xmark-sharp-regular-1" class="h-5 mx-3">
```
Comment (edithemmings) [inline app/views/layouts/_flashes.html.erb:19]: Generally, interpolation is preferred over concatenation
https://patrickkarsh.medium.com/interpolation-and-concatenation-7c9dbda6da64

```suggestion
        <img src="<%= asset_path("flashes/#{name}.svg") %>" alt="xmark-sharp-regular-1" class="h-5 mx-3">
```

File: app/views/layouts/_flashes.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                when 'error'
+                  'red'
+                when 'warning'
+                  'yellow'
+                when 'info'
+                  'blue'
+                end %>
+  <div class="fixed right-0 top-0 mx-2 my-2 w-2/5">
+    <!-- Narrow dark box on top -->
+    <div class="h-2 rounded-t bg-<%= color %>-500"></div>
```
Comment (edithemmings) [inline app/views/layouts/_flashes.html.erb:14]: OPTIONAL:
Another way of writing a case-switch like this to create an object and fetch the value you need by passing in the key. For example:
```
<%  flash_class = {
    'success' => 'bg-green-500',
    'error' => 'bg-red-500',
  ....
  } %>
```
```
<div class="h-2 rounded-t <%= flash_class[name] %>"></div>
```

**Review by edithemmings — COMMENTED** [review]
Awesome work!! Great job working off of the mockups and putting together a very concise, clean PR. I had to get picky here because there wasn't much to say. And I see you have the approval, so if you're ready to go ahead and merge, feel free. Hopefully my comments are at least helpful for your learning.

**Review by edithemmings — APPROVED** [review]
🎉🚀

**Pull request merged**